src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -569,161 +559,133 @@
+          "description": "Key/value pairs that provide additional information about the rule.",
           "type": "object",
           "default": {}
         },
 
-        "tags": 
-        {
-          "description": "A unique set of strings that provide additional information for the associated rule.",
+        "tags": {
+          "description": "A set of distinct strings that provide additional information about the rule.",
           "type": "array",
-          "items":
-          {
+          "items": {
             "type": "string",
-            "default": []
+            "default": [ ]
           }
         }
       },
       "required": ["id"]
     },
     "run": 
     {
-      "description": "A run object describes the invocation of the static analysis tool that produced the results specified in the containing runLog object. NOTE: The information in the run object makes it possible to precisely repeat a run of a static analysis tool, and to verify that the results reported in the log file were generated by an appropriate invocation of the tool.",
+      "description": "Describes a single run of a static analysis tool, and contains the output of that run.",
       "additionalProperties": false,
       "type": "object",
       "properties": {
-        "invocation": 
-        {
-          "description": "A string that describes any parameterization for the tool invocation. For command line tools this string may consist of the completely specified command line used to invoke the tool.",
+        "tool": {
+          "description": "Information about the tool or tool pipeline that generated the results in this log. A results log can only contain results produced by a single tool or tool pipeline. A results log can aggregate results from multiple tool log files, as long as context around the tool run (tool command-line arguments and the like) is identical for all aggregated files.",
+          "$ref": "#/definitions/tool"
+        },
+
+        "invocation": {
+          "description": "A string containing the runtime parameters with which the tool was invoked. For command line tools, this string may consist of the completely specified command line used to invoke the tool.",
           "type": "string"
         },
 
-        "files": 
-        {
-          "description": "A dictionary each of whose keys is a URI and each of whose values is an array of fileReference objects representing the location of a single file target scanned during the run.",
+        "files": {
+          "description": "A dictionary each of whose keys is a URI and each of whose values is an array of file objects representing the location of a single file target scanned during the run.",
```

Comment:
Can't see which word you're referring to; I assume you're asking whether I wanted "A dictionary, each of whose keys..." *ponders* I don't think it's strictly necessary, and I liked my restraint because I tend to overuse commas... but... ok, sure.